tests/Tests/600__ai_apps/660__openvino/660__openvino.robot

```diff
@@ -22,6 +22,29 @@ Verify OpenVino Is Available In RHODS Dashboard Explore Page
   Verify Service Is Available In The Explore Page    OpenVINO
   Verify Service Provides "Get Started" Button In The Explore Page    OpenVINO
 
+Verify OpenVINO Image Is Tracked By Notification System In RHODS Dashboard
+    [Tags]    Tier2
+    ...       ODS-652
+    [Documentation]  Cheks whether RHODS notification shows the Notebook images are running after installing openvino
+    Check And Install Operator in Openshift    ${openvino_operator_name}   ${openvino_appname}
+    Create Tabname Instance For Installed Operator        ${openvino_operator_name}       Notebook    redhat-ods-applications
+    Wait Until Keyword Succeeds    900  1     Check Image Build Status   Complete     openvino-notebook
+    Get Build Status    namespace=redhat-ods-applications    build_search_term=openvino-notebook
+    Click Element    //a[@class="co-resource-item__resource-name"]
+    Wait Until Page Contains    Build details
+    @{link_elements}=  Get WebElements
+    ...    //div[@class='co-m-label-list']
+    @{list} =    Create List
+    FOR  ${link}  IN  @{link_elements}
+          ${txt} =    Get Text    ${link}
+          Append To List    ${list}    ${txt}
+    END
+    Run Keyword And Return Status    Should Contain    ${list}[0]    opendatahub.io/build_type\n=\nnotebook_image
+    Go To RHODS Dashboard
+    Click Button    //*[@id="root"]/div/header/div[2]/div/div[1]/button    #bell icon
+    Wait Until Page Contains    Notebook images are running
+    [Teardown]   Uninstall Openvino Operator
+
```

Comment:
could you please integrate this TC into `Verify Openvino Operator Can Be Installed Using OpenShift Console`? Otherwise we'll install, deploy and uninstall openvino twice in a run. ODS-652 is small enough to be implemented inside the already existent one

Reply:
Done